src/Symfony/Component/Validator/Resources/translations/validators.ar.xlf

```diff
@@ -124,15 +124,15 @@
             </trans-unit>
             <trans-unit id="34">
                 <source>The file could not be uploaded.</source>
-                <target>لم استطع استقبال الملف.</target>
+                <target>تعذر تحميل الملف.</target>
             </trans-unit>
             <trans-unit id="35">
                 <source>This value should be a valid number.</source>
                 <target>هذه القيمة يجب ان تكون رقم.</target>
             </trans-unit>
             <trans-unit id="36">
                 <source>This file is not a valid image.</source>
-                <target>هذا الملف ليس صورة صحيحة.</target>
+                <target>هذا الملف غير صالح للصورة.</target>
```

Comment:
>While the change aligns better with formal language, it may lose some clarity depending on the user's familiarity with technical terms like "غير صالح للصورة" (invalid for an image).